README.md

```diff
@@ -82,7 +82,7 @@ For more information on how cursor-based pagination works, and how it's standard
 ### Examples
 
 ```ts
-import { findWithPagination } from "@kazoohr/typeorm-pagination";
+import { findWithPagination } from "@WorkTangohr/typeorm-pagination";
```

Comment:
Hah I know this isn't quite what it looks like in the code, but also maybe we should keep the lowercasing? 🤷 

```suggestion
import { findWithPagination } from "@worktango/typeorm-pagination";
```

Reply:
I switched it back to kazoohr for consistency